deploy/csv-templates/ics-operator.csv.yaml.in

```diff
@@ -394,6 +446,8 @@ spec:
           - patch
           - update
           - watch
+          - deletecollection
```

Comment:
@malayparida2000 Do you know of any places the OCS operator needs to delete multiple resources in a single API call? If not, the `deletecollection` would not be needed to add when removing the wildcard.

Reply:
I dont beleive if we are deleting collections anywhere in the ocs-operator